WordPress/src/main/java/org/wordpress/android/ui/sitecreation/NewSitePreviewViewModel.kt

```diff
@@ -168,6 +173,7 @@ class NewSitePreviewViewModel @Inject constructor(
             }
             FAILURE -> {
                 serviceStateForRetry = event.payload as NewSiteCreationServiceState
+                tracker.trackErrorShown(ERROR_CONTEXT, "unknown", "NewSiteCreation service failed")
```

Comment:
Can we add a constant for the error type and message instead of hardcoding here? I'd suggest adding them in the `NewSiteCreationTracker` maybe as an `enum` or something so it's a bit more structured. What do you think?

Reply:
> It's basically just a log message and it's used just once.

I don't think re-usability is the only concern when we put `String`s into constants. I think it's still important to keep the code clean, but this is a nitpick, so it's up to you.